common/src/main/java/xyz/wagyourtail/jsmacros/client/api/sharedclasses/PositionCommon.java

```diff
@@ -295,6 +381,10 @@ public BlockPosHelper toBlockPos() {
         public BlockPos toRawBlockPos() {
             return new BlockPos(Math.floor(x), Math.floor(y), Math.floor(z));
         }
+        
+        public Vec3d toRawVec3d() {
```

Comment:
idk about the name of this one... seems confusing
maybe, `toRawMcVector` or something

Reply:
I renamed it to `toMojangDoubleVector` inspired by the `toMojangFloatVector` method in Vec3D.